InterfacePython/TasmanianSG.py

```diff
@@ -50,6 +50,33 @@
 lsTsgLocalRules = ["localp", "semi-localp", "localp-zero", "localp-boundary"]
 lsTsgAccelTypes = ["none", "cpu-blas", "gpu-default", "gpu-cublas", "gpu-cuda", "gpu-rocblas", "gpu-hip", "gpu-magma"]
 
+def getTsgLib(tasmanian_library=0):
+    '''
+    tasmanian_library: indicates the libtasmaniansparsegrids.so
+
+        if tasmanian_library is an int:
+            look for the library in the default location "libtasmaniansparsegrid.so" or the library install prefix given to CMake
+
+        if tasmanian_library is a string:
+            use it as a path; for example, TasmanianSparseGrid(tasmanian_library = "opt/Tasmanian/lib/libtasmaniansparsegrid.so")
+
+        otherwise:
+            tasmanian_library must be an instance of ctypes.cdll this is useful when creating lots of instances of TasmanianSparseGrid
+            in order to avoid having each instance load a sepatate copy of the common library
+    '''
+    if (isinstance(tasmanian_library, int)):
```

Comment:
In Python, if you want to have a dummy parameter, do not use `int` use `None`.

Reply:
Removed this line as part of the fix for the above issue.